src/effects/VST3/VST3Effect.cpp

```diff
@@ -406,20 +406,23 @@ bool VST3Effect::LoadUserPreset(
    if(!mEditController)
       return false;
 
-   if(!PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, wxT("ProcessorState")))
+   constexpr auto processorStateKey  = wxT("ProcessorState");
+   constexpr auto controllerStateKey = wxT("ControllerState");
+
+   if(!PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, processorStateKey))
       return false;
 
    wxString processorStateStr;
-   if(!GetConfig(*this, PluginSettings::Private, name, wxT("ProcessorState"), processorStateStr, wxEmptyString))
+   if(!GetConfig(*this, PluginSettings::Private, name, processorStateKey, processorStateStr, wxEmptyString))
       return false;
    auto processorState = PresetsBufferStream::fromString(processorStateStr);
    if(mEffectComponent->setState(processorState) != kResultOk)
       return false;
 
-   if(PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, wxT("ProcessorState")))
+   if(PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, controllerStateKey))
    {
       wxString controllerStateStr;
-      if(!GetConfig(*this, PluginSettings::Private, name, wxT("ControllerState"), controllerStateStr, wxEmptyString))
+      if(!GetConfig(*this, PluginSettings::Private, name, controllerStateKey, controllerStateStr, wxEmptyString))
```

Comment:
The elimination of duplicated string constants is incomplete.  The new names should also be used in `SaveUserPreset`.  That can help increase confidence that there is not a mismatch between the save procedure, which left the data in the config file for the load procedure to interpret here.